server/config.go

```diff
@@ -61,12 +61,13 @@ import (
 
 // Config represents the configuration format for the kapacitord binary.
 type Config struct {
-	HTTP           httpd.Config      `toml:"http"`
-	Replay         replay.Config     `toml:"replay"`
-	Storage        storage.Config    `toml:"storage"`
-	Task           task_store.Config `toml:"task"`
-	InfluxDB       []influxdb.Config `toml:"influxdb" override:"influxdb,element-key=name"`
-	Logging        logging.Config    `toml:"logging"`
+	HTTP     httpd.Config      `toml:"http"`
+	Replay   replay.Config     `toml:"replay"`
+	Storage  storage.Config    `toml:"storage"`
+	Task     task_store.Config `toml:"task"`
+	InfluxDB []influxdb.Config `toml:"influxdb" override:"influxdb,element-key=name"`
+	// TODO(desa): this should change to diagnostic eventually
+	Logging        diagnostic.Config `toml:"logging"`
```

Comment:
This config is about logging, but logging is implemented through diagnostics so I think this name is fine.

Reply:
fixed